app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -97,7 +97,7 @@ To find out about teaching in other areas of the UK, visit the:
 * [Teach in Scotland website](https://teachinscotland.scot/)
 * [Northern Ireland Department for Education website](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland)
 
-If you don’t want to come to England to study you can apply to train for international qualified teacher status (iQTS). This meets the same high standards as [English qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) and, if successfully completed, leads to the automatic award of QTS.
+If you don’t want to come to England to study you can apply to train for international qualified teacher status (iQTS). This meets the same high standards as [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) and, if successfully completed, leads to the automatic award of QTS.
```

Comment:
Should we really have three links to the same page? It's not best practice in terms of accessibility (although I appreciate the page is very long!)

Will leave you to decide @joegibb @Sarah-DfE !